src/ex4/dist/clients/item-client.js

```diff
@@ -0,0 +1,67 @@
+// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
+const SERVER_URL = 'http://localhost:8080/todo'
+export default class ItemClient {
+
+    async getTodoList(query = ""){
+        const response = await fetch(`${SERVER_URL}${query}`)
+        const data = await response.json()
+
+        return data
+    }
+
+    async addTodo(value) {
+        await fetch(SERVER_URL, 
+        {
+            method: 'POST',
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({todo: value})
+        })
+    }
+
+    async deleteTodo(index){
```

Comment:
working with indexes instead of ids could lead to many bugs, consider switching to ids.